packages/web-shared/components/Searchbar/Search.styles.js

```diff
@@ -83,6 +83,15 @@ const Wrapper = withTheme(styled.div`
   .aa-Item {
     border-radius: 0;
   }
+
+  .chip-list-feature {
+    * > &:first-child {
+      margin-left: 1rem;
+    }
+    * > &:last-child {
+      margin-left: 1rem;
+    }
+  }
```

Comment:
Nice!

Reply:
Thanks! This change was functional as well - I needed to make sure the margin wouldn't show up on the regular feed, just in the context for search.